iphone/Classes/TiUIScrollableViewProxy.m

```diff
@@ -71,7 +71,7 @@ -(NSUInteger)viewCount
 
 -(void)setViews:(id)args
 {
-#ifndef TI_USE_AUTOLAYOUT
+#ifdef TI_USE_AUTOLAYOUT
```

Comment:
Since this is reverted   `ifndef`  to ` ifdef`  in the versions 5.1.2 and 5.2.RC now the RE `setView()` freez the app.

I just tested now with` ifndef  `and the app not freez anymore.